node-graph/gcore/src/registry.rs

```diff
@@ -9,7 +9,7 @@ use std::sync::{LazyLock, Mutex};
 pub use graphene_core_shaders::registry::types;
 
 // Translation struct between macro and definition
-#[derive(Clone)]
+#[derive(Clone, Debug)]
```

Comment:
Intentional?

Reply:
Yes, I needed this for debugging and there is not really a good reason to not have this derive debug